### PR TITLE
userfeedback initial commit

### PR DESCRIPTION
**Pull request by angeloremmilbautista**
Developed user feedback API endpoint

https://trello.com/c/G7NqAExo

### CODE REVIEW

File: controller/userFeedbackController.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+let addUserFeedback = require('../model/addUserFeedback.js')
+
+const userfeedback = async (req, res) => {
+  try {
+    const { name, contact_number, email, experience, comments } = req.body;
+    if (!name) {
+      return res.status(400).send('Name is required');
```
Comment (sicsixdkn) [inline controller/userFeedbackController.js:7]: Can we standardise all messages sent to be { message: <message> }

File: controller/userFeedbackController.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    if (!comments) {
+      return res.status(400).send('Comments is required');
+    }
+    
+    await addUserFeedback(name, contact_number, email, experience, comments)
+
+    res.status(201).json({ message: 'Data received successfully!' });
+  } catch (error) {
+    console.error(error);
+    res.status(500).json({ message: 'Internal server error' });
```
Comment (sicsixdkn) [inline controller/userFeedbackController.js:27]: Also, for errors can we use the format { error: <error> }

**Review by sicsixdkn — CHANGES_REQUESTED** [review]
Just a couple of changes to standardise the message/error object format, will make the changes elsewhere to match.

Working on an API spec in postman and need to get it all matched up.

Looks good!

File: controller/userFeedbackController.js (hunk 1 above)
Comment (angeloremmilbautista) [reply controller/userFeedbackController.js:7]: Resolved

**Review by sicsixdkn — APPROVED** [review]

**Pull request merged**